go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/devfile/library
 
-go 1.14
+go 1.13
```

Comment:
Any reason why we're downgrading to 1.13 here and the other mods?

Reply:
Since api/odo/console/workspace all use go 1.13.  I want to ensure our library is also using the same go version,to be consistent with the api library and our consumers.